src/client/components/ProductsView/ProductsView.component.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import './ProductsView.styles.css';
 import Product from '../Product/Product.component';
+import PropTypes from 'prop-types';
 
```

Comment:
Import offerProductsModal from specialOffers component. and use it in the map function.